doc/source/dev/conduct/enforcement.md

```diff
@@ -0,0 +1,211 @@
+# SciPy Code of Conduct - Enforcement Manual
+
+This is the enforcement manual followed by SciPy's Code of Conduct
+Committee. It's used when we respond to an issue to make sure we're consistent
+and fair.
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee"), currently comprised of
+[*note:* still gathering volunteers, this will be filled in as they become
+available and prior to final upload into the rendered site]:
+
+* Person One foo@email (Chair of the Committee)
```

Comment:
Will publishing emails like this mean that if someone has an outdated scipy version (and outdated docs), then it's entirely possible that the whole committee could have changed?